### PR TITLE
Put status in Purchase instead of Order. Added a getPurchasesByVendor…

### PR DESCRIPTION
**Pull request by Huntrr**
… method to Order

### CODE REVIEW

File: app/models/purchase.py [hunk 1]
```diff
@@ -57,12 +57,28 @@ def __init__(self, cart_items):
     def __repr__(self):
         return "<Order: {}>".format(self.id)
 
+    def getAllPurchases(self):
```
Comment (jondubin) [inline app/models/purchase.py]: Nit: can you make these function names `snake_case`

File: app/models/purchase.py [hunk 2]
```diff
@@ -57,12 +57,28 @@ def __init__(self, cart_items):
...
+        return Purchase.query.filty_by(order_id=self.id)
```
Comment (jondubin) [inline app/models/purchase.py]: `filty` should be `filter` and shouldn't you return a list of purchases a la `.all()`?

File: app/models/purchase.py [hunk 3]
```diff
@@ -57,12 +57,28 @@ def __init__(self, cart_items):
     def __repr__(self):
         return "<Order: {}>".format(self.id)
 
+    def get_all_purchases(self):
```
Comment (jondubin) [inline app/models/purchase.py:60]: Why do we need a `get_all_purchases` function? `{order}.purchases` should work the same

**Comment by jondubin** [comment]
@Huntrr I went ahead and made the changes so that I can use them in the order view

**Comment by jondubin** [comment]
@Huntrr with the most recent changes in master, this PR can be closed

**Comment by Huntrr** [comment]
@jondubin Fine by me. Closing PR. Leave the branch?

**Comment by jondubin** [comment]
Branch can also go

**Pull request closed without merging**